src/plugins/mini/README.md

```diff
@@ -132,7 +132,8 @@ kdb lsmeta /examples/mini
 kdb get /examples/mini/key
 #> "multi
 
-# Undo modifications to the key database
+# Undo modifications
+rm stderr.txt
```

Comment:
Is the current working directory something specified by shell recorder? (please add to docu) If not, it should be and the shell recorder should track that every file is removed from this folder.

Reply:
> Is the current working directory something specified by shell recorder?

Yes, the working directory is the [root folder of the repository](https://github.com/ElektraInitiative/libelektra/commit/e5b89c46).

> …the shell recorder should track that every file is removed from this folder.

I agree. For now, I created a ToDo item about this issue.